acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/cms/ps/CmsSteps.java

```diff
@@ -547,13 +547,8 @@ public void iCanCopyThePublication() throws Throwable {
         contentPage.copyDocument();
     }
 
-    @Then("^I can preview the document$")
```

Comment:
Hmm, given that this is a 'Then' clause, the 'can' made sense to indicate that this is an assertion rather than action.

Reply:
I'll change it to @When as this is what it actually is now, it doesn't assert anything anymore